internal/uvm/clone.go

```diff
@@ -0,0 +1,101 @@
+package uvm
+
+import (
+	"context"
+
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
+	"github.com/pkg/errors"
+)
+
+const (
+	hcsSaveOptions = "{\"SaveType\": \"AsTemplate\"}"
```

Comment:
Prefer using a raw string here to avoid having to escape things. Also if this is the only constant we don't need the multi variable syntax of ().

```go 
const hcsSaveOptions = `{"SaveType": "AsTemplate"}`
```